infra/app/api-center.bicep

```diff
@@ -0,0 +1,13 @@
+param name string
```

Comment:
@dotnet-7 - Let's get a PR to azd/infra/core for this...and move this file to infra/core/api folder.

@justinyoo - Do you think api center falls under "gateway", "networking", or should we create a new "api" folder?

Reply:
@jongio I would go for the "gateway" because of APIM. Alternatively, we can create a new folder like "integration". But for now, I'd more lean on the "gateway" folder.